go-controller/pkg/node/gateway_init_linux_test.go

```diff
@@ -1174,7 +1174,7 @@ OFPT_GET_CONFIG_REPLY (xid=0x4): frags=normal miss_send_len=0`,
 					"-s 169.254.169.1 -j ACCEPT",
 					"-d 172.16.1.0/24 -j ACCEPT",
 					"-s 172.16.1.0/24 -j ACCEPT",
-					"-o ovn-k8s-mp0 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT",
+					"-o ovn-k8s-mp0 -j ACCEPT",
 					"-i ovn-k8s-mp0 -j ACCEPT",
```

Comment:
```suggestion
					"-i ovn-k8s-mp0 -j ACCEPT",
					"-o ovn-k8s-mp0 -j ACCEPT",
```

Following unit tests fails because [iptables.AddRules(..., false)](https://github.com/ovn-org/ovn-kubernetes/blob/2d4ea07f2cf2b058ae59573d2b07ccb96ed8a090/go-controller/pkg/node/iptables/iptables.go#L21) insert iptables in a reverse order than the input slice.

```
[Fail] Gateway Init Operations Setting up the gateway bridge [It] sets up a local gateway with predetermined interface 
/home/runner/work/ovn-kubernetes/ovn-kubernetes/go-controller/pkg/node/gateway_init_linux_test.go:1195

[Fail] Gateway Init Operations Setting up the gateway bridge [It] sets up a local gateway with predetermined interface and no default route 
/home/runner/work/ovn-kubernetes/ovn-kubernetes/go-controller/pkg/node/gateway_init_linux_test.go:1195
```

Reply:
yeah hehe my bad, I was in a hurry thursday evening; I have fixed this. will open a d/s merge and see how this works on migration